src/DocConfiguration.ts

```diff
@@ -0,0 +1,164 @@
+import express from 'express';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+
+const swaggerDefinition = {
+    openapi: '3.0.0',
+    info: {
+        title: 'RIF Relay Server API',
+        version: '1.0.0',
+        description: 'This is a API application provided to support RIF Relay.',
+        license: {
+            name: 'Licensed Under MIT',
+            url: 'https://spdx.org/licenses/MIT.html'
+        },
+        contact: {
+            name: 'RSK Smart',
+            url: 'https://developers.rsk.co/'
+        }
+    },
+    servers: [
+        {
+            url: 'http://localhost:8090',
+            description: 'Development server'
+        }
```

Comment:
We may need to add further servers (testnet/mainnet). Maybe we could add the local server just if we are running locally. WDYT?

Reply:
I ended up with another solution: we may just use one single server and the URL would be taken directly from the configuration. WDYT?

Reply:
that's neat! 

i checked the openAPI examples and found the following:

```json
{
  "servers": [
    {
      "url": "https://development.gigantic-server.com/v1",
      "description": "Development server"
    },
    {
      "url": "https://staging.gigantic-server.com/v1",
      "description": "Staging server"
    },
    {
      "url": "https://api.gigantic-server.com/v1",
      "description": "Production server"
    }
  ]
}
```  

so while the running URL is definitely useful, i think we should still keep in mind the idea of adding the testnet/mainnet URLs when they are available.

so maybe a good middle ground would be the running server + these 2? i guess we might have a problem if they are repeated though :thinking:

Reply:
you've convinced me :+1: 

running servers can be listed elsewhere, or if the `docs` endpoint is made available publicly.